cmd/egnkey/derive_operatorid/derive_operatorid.go

```diff
@@ -0,0 +1,39 @@
+package derive_operatorid
```

Comment:
probably want to avoid this package name 

https://go.dev/blog/package-names

```
Good package names are short and clear. They are lower case, with no under_scores or mixedCaps. They are often simple nouns, such as:

time (provides functionality for measuring and displaying time)
list (implements a doubly linked list)
http (provides HTTP client and server implementations)
The style of names typical of another language might not be idiomatic in a Go program. Here are two examples of names that might be good style in other languages but do not fit well in Go:

computeServiceClient
priority_queue
```

Reply:
changed to `operatorid`: [01fc8ae](https://github.com/Layr-Labs/eigensdk-go/pull/247/commits/01fc8aec9c512f1f4b5dfb98095e2765a21145d2)